svc/ClientHealthService.ts

```diff
@@ -64,6 +61,25 @@ export class ClientHealthService extends HoistService {
         this.sources.delete(key);
     }
 
+    /**
+     * Generate and submit a report to the server, via TrackService.
+     * @internal - apps should enable via config and allow this service to submit on timer.
+     */
+    sendReport() {
+        const {intervalMins, ...rest} = XH.trackService.conf.clientHealthReport ?? {};
+
+        XH.track({
+            category: 'App',
+            message: 'Submitted health report',
+            ...rest,
```

Comment:
our strategy of having the `..rest` coming from config including "debug" not going to be compatible with this. :-(

Reply:
Good catch - worked around it with...yet more opts....

Reply:
maybe their is a public async version of this method that sets the severity to 'info', and awaits the pending message push.  And a private version called by that and the timer....

Reply:
FWIW now that I've added opts, we could bring in the `sendImmediately` and `showConfirmToast` etc.  Could also support passing in custom data for that one-off call.  

But still didn't think any of those were quite compelling enough to add up-front, and trying to keep as minimal as possible.  

The entire feature is clearly a bonus - was triggered from desire to see report from mobile client without waiting 15 mins, and without having dev console to trigger manually